react/features/stream-effects/blur/index.js

```diff
@@ -1,8 +1,10 @@
 // @flow
 
 import { load } from '@tensorflow-models/body-pix';
-
+import * as TF from '@tensorflow/tfjs';
 import JitsiStreamBlurEffect from './JitsiStreamBlurEffect';
+import { getLogger } from 'jitsi-meet-logger';
+const logger = getLogger(__filename);
```

Comment:
Please don't use the logger here. Since it's a standalone bundle it will just increase the bundle size for very little gain. Just use console. Elsewhere in the code, however, do use it! :-)

Reply:
I switched to using the console here. Also, figured out another way, i.e., importing just the tfjs-core for setting the backend which seems to have reduced the bundle size significantly.